PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -456,7 +456,9 @@ extension ArchivedItemsListViewModel {
                 source: source,
                 tracker: tracker.childTracker(hosting: .myList.myList),
                 listOptions: listOptions,
-                sender: sender
+                sender: sender,
+                sourceView: .archiveList,
```

Comment:
curious - what's the significance of using this `sourceView`? Do we still need it, it seems like `listOptions` has the useful data that we need.

Reply:
@cyndichin You are right, this needs to be removed/refactored. We applied iterated solutions before optimizing ListOptions completely and 1 of the solution had sourceView as a dependency for SortMenu to identify which property needs to be modified and notified. However, with the new ListOptions code/logic, sourceView isn't needed anymore. This is removed. thank you.

Reply:
thanks for the background context on this!